### ISSUE TITLE
Icon update: fa-question

### ISSUE THREAD

**Issue by avcoder** [FortAwesome/Font-Awesome#19239]
Hello Font Awesome

Is it possible for you to thicken the solidness of 'fa-question'.
Currently even with 'fas' applied, the question mark still looks too thin.

![image](https://user-images.githubusercontent.com/7874705/182875269-aceafa4d-ed5a-4945-a0f8-cf9fbef4ddd1.png)

**Comment by sensibleworld** [comment]
The issue is that the question mark thickness is now consistent with other "line-based" icons like `bars` `arrow`, etc. I could could see making a thicker variant, though.

For your use-case, though, would `circle-question` work?

**Comment by tagliala** [comment]
@sensibleworld it is also consistent with `I`, `1`, `!`, and other(s)

**Comment by sensibleworld** [comment]
@tagliala Yup. That's primarily why we made the change. But I can see a use case for thicker icons. Not to replace the current versions, but in addition to them. Kind of like the below:

<img width="911" alt="CleanShot 2022-08-04 at 16 15 29@2x" src="https://user-images.githubusercontent.com/529604/182944796-d86ae363-5be0-42d9-89ba-7263d78608c9.png">
